### PR TITLE
Fix installer for RHEL7, broken on Docker verification with "No package docker-io available."

### PR DESCRIPTION
**Pull request by marcellodesales**
Problem
----

The current installer assumes the Docker RPM module is named `docker-io`. However, the name has been renamed to `docker-engine`. The installation halts with the following error message:

```sh
Nothing to do
No package docker-io available.
Error: Nothing to do
oh no! something went wrong during the install...
contact support@npmjs.com and we can help get you up and running
```

Solution
---- 

Update the script to properly rename the package during the installation. I could only properly install and setup the registry on RHEL 7 with this patch. Finally I have it on RHEL 7.

```sh
Complete!
Congrats! Your npm On-Site server is now up and running \o/

There are just a few final steps:

Step 1. Access this server in a web-browser via port 8800 (this will bring you to an admin console)
Step 2. Proceed passed the HTTPS connection security warning (a selfsigned cert is being used initially)
Step 3. Upload a custom TLS/SSL cert/key or proceed with the provided self-signed pair.
Step 4. Configure your npm instance & click "Save".
Step 5. Visit https://docs.npmjs.com/, for information about using npm On-Site or contact support@npmjs.com
```

### CODE REVIEW

File: bin/npmo.js [hunk 1]
```diff
@@ -58,6 +58,13 @@ function install (yargs) {
         return
       }
 
+      // [npme@pppdc9prda2x npmo]$ cat /etc/redhat-release
+      // RHEL7: Red Hat Enterprise Linux Server release 7.1 (Maipo)
+      // CENTOS7: CentOS Linux release 7.0.1406 (Core)
+      var redHatOS = fs.readFileSync('/etc/redhat-release').toString()
```
Comment (nexdrew) [inline bin/npmo.js]: This isn't going to work for platforms other than RHEL/CentOS. :wink:

**Comment by nexdrew** [comment]
I wonder if this is still necessary? I went through CentOS 7 and RHEL 7 install tests today ([notes here](https://github.com/npm/npmo-installer/issues/87#issuecomment-159413777)) and did not encounter this same problem.

File: bin/npmo.js (hunk 1 above)
Comment (marcellodesales) [inline bin/npmo.js]: Oops, sorry!

**Comment by marcellodesales** [comment]
If RHEL 7 works, I'm fine closing this out...

**Pull request closed without merging**